src/actions/blogList.js

```diff
@@ -10,9 +10,9 @@ export const blogUpvote = (id) => (dispatch, getState) => {
     axios.get(serverData.django_server + `/api/blog/${id}/upvote/`, tokenConfig(getState))
         .then(res => {
             if (res.data === "already_voted")
-                alert("You've already voted!");
+                dispatch(createMessage({vote: "You have already Up-Voted"}));
```

Comment:
We are now sending this message from the backend.

Reply:
It's been changed